release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1087,6 +1089,8 @@ In {product-title} 4.13, support for {op-system} functionality is removed for th
 [id="ocp-4-13-bare-metal-hardware-bug-fixes"]
 ==== Bare Metal Hardware Provisioning
 
+* Previously, when you attempted to deploy an {product-title} cluster node on a server that is configured with the Integrated Lights-Out (iLO) Management Interface Driver, provisioning of the node would fail. You would receive an error message similar to "An auth plugin is required to determine endpoint URL". This issue occurs because of a missing `[ilo]/use_web_server_for_images` configuration parameter in the iLO driver that causes the driver to attempt to use object storage as the default storage mechanism. Object storage is not present in the product. The {product-title} 4.13 version and later versions fix this issue by including `[ilo]/use_web_server_for_images` in the iLO driver's configuration, which ensures that the iLO driver uses a web server running in the metal3 pod. (link:https://issues.redhat.com/browse/OCPBUGS-5068[*OCPBUGS-5068*])
```

Comment:
"Deploying" would not work because the dependant clause also begins with a gerund, "provisioning".

Guidance used to exist about using "because" over "due", so I used "because" to be safe.

I used "present" in the verb form so I retained the "in the product" clause. 

Removed "fixing the failire2 as it dangles a bit.

------------------
All other changes implemented.